lib/jekyll-github-metadata.rb

```diff
@@ -1,4 +1,9 @@
 require 'octokit'
+begin
+  require 'dotenv'
```

Comment:
Sure, sounds good! But i think we should put `require 'jekyll'` also in the  `begin` block  then because it was removed in https://github.com/jekyll/github-metadata/commit/80370fc0c04198bc6e5e3cdce4966e9a17dbbb73 and it's only a development dependency in the gemspec